_docs/_latest/configure/gsuite-group-collection.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Enabling GSuite Google Groups Collection
-order: 204
+order: 600
```

Comment:
Please shorten this title to "GSuite" as well as the filename.

Reply:
Updated